src/constraints/storage.jl

```diff
@@ -100,36 +102,37 @@ function add_storage_constraints!(
             profile_aggregation(
                 Statistics.mean,
                 graph[row.asset].rep_periods_profiles,
+                row.year,
                 ("min_storage_level", row.rep_period),
                 row.timesteps_block,
                 0.0,
             ) * (
-                graph[row.asset].initial_storage_capacity +
-                (row.asset ∈ Ai ? energy_limit[row.asset] : 0.0)
```

Comment:
@datejada I also came across this: the same `energy_limit` is used for both max and min constraints?

Reply:
This is correct since we use a minimum storage level profile for the min constraint 😄